packages/components/src/Card/Card.css

```diff
@@ -56,6 +56,12 @@
 }
 
 .header {
+  display: flex;
+  flex-direction: row;
   padding: var(--card--base-padding);
   background-color: var(--color-surface--background);
 }
+
+.headerButton {
```

Comment:
Minor but I think this should be renamed to `headerAction` to match the prop names